### DISCUSSION TITLE
Why is there no function create_rw_slice?

### DISCUSSION THREAD

**Discussion by ghost** [leptos-rs/leptos#2546] [Q&A] [answered]
Hello and thanks for the best framework!
I'm on the stable rust. Therefore, i prefer to create signals by calling `create_rw_signal`. But there is no similar function for slices.
Is there any special reason for this?

**Comment by gbj** [comment, marked as answer]
See related discussion in #2153 and #2356. It would be very easy to create your own `create_rw_slice` if you'd like (< 20 LOC) — [The code for `create_slice`](https://docs.rs/leptos_reactive/0.6.11/src/leptos_reactive/slice.rs.html#72-78) is very simple and you could just create a struct that does a similar thing,.
